app/main.py

```
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+    src, dst = parts[1], parts[2]
+    if src == dst:
+        return
+    try:
+        with open(src, "r") as file_in, open(dst, "w") as file_out:
+            file_out.write(file_in.read())
+    except FileNotFoundError:
+        pass
```

Silently passing on FileNotFoundError may not be ideal if the requirements specify that the user should be informed of errors. If error reporting is required by the checklist, you should handle this differently.